util/worker.cpp

```diff
@@ -427,6 +429,71 @@ ConcreteVal *WorkerInterpreter::loadConcreteVal(const IR::Type &type,
   return nullptr;
 }
 
+
+ConcreteBlock WorkerInterpreter::loadConcreteBlock(const ojson &block, bool init) {
+  ConcreteBlock c_block;
+  c_block.size = block["size"].as_integer<uint64_t>();
+  c_block.address = 0; // TODO where would this be used
+  c_block.align = block["align"].as_integer<uint64_t>();
+  
+  if (c_block.size == 0 || !block.contains("bytes")) {
+    return c_block;
+  }
+  
+  for (auto& json_byte : block["bytes"].array_range()) {
+    assert(json_byte.size() == 3 && "each byte contains 3 elements");
+    if (json_byte[0].is_null()) {
```

Comment:
You should load this byte just like the other bytes. You can make the value byte / ptr byte code below create the ConcreteByte, without emplacing it yet. Then add code after that to set `c_block.default_byte` or call `c_block.bytes.emplace` depending on whether `json_byte[0].is_null()` is true.

Reply:
I think I mistakenly assumed that the default byte is always the poison byte based on the example test files.
i.e. [null, 0, 0]. to confirm, the default byte can be any byte value?